crypto/x509/x_x509.c

```diff
@@ -116,6 +116,14 @@ ASN1_SEQUENCE_ref(X509, x509_cb) = {
 IMPLEMENT_ASN1_FUNCTIONS(X509)
 IMPLEMENT_ASN1_DUP_FUNCTION(X509)
 
+void X509_set0_libctx(X509 *x509, OPENSSL_CTX *libctx, const char *propq)
```

Comment:
If we're going to take this approach, I wonder if we need the newly added `X509_verify_ex()` function. But so far I'm wondering why we need it at all...

Reply:
Removed the function.